test/test_utils/__init__.py

```diff
@@ -1265,6 +1289,21 @@ def get_synapseai_version_from_tag(image_uri):
     return synapseai_version
 
 
+def get_synapseai_version_from_tag(image_uri):
```

Comment:
this is a duplicate function. we should remove this from private repo __init__.py file

Reply:
remove from PR, I see only one `get_synapseai_version_from_tag` in private repo [here](https://github.com/aws/PRIVATE-deep-learning-containers/blob/private-master/test/test_utils/__init__.py#L1239).